deps/valgrind/valgrind.h

```diff
@@ -6139,7 +6139,7 @@ typedef
           /* Memory pool support. */
           VG_USERREQ__CREATE_MEMPOOL   = 0x1303,
           VG_USERREQ__DESTROY_MEMPOOL  = 0x1304,
-          VG_USERREQ__MEMPOOL_ALLOC    = 0x1305,
+          VG_USERREQ__MEMSMALL_ALLOC    = 0x1305,
```

Comment:
Not sure this particular change makes sense, considering the other memory pool related definitions.